Helpers/opent2t-translator-helper-hue/js/index.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
```

Comment:
This should follow the convention of "hubs" as "providers" that the Nest and Wink translators currently use.  PRobably a com.hue.hub translator that can then be used by the device translators.  See Arjun's recent changes for Nest as an example.  Nest, like Hue, doesn't have a physical hub, so in this case the hub is a representation of a single API/service that is used to talk to the devices.